packages/display/src/Container.ts

```diff
@@ -573,23 +576,71 @@ export class Container extends DisplayObject
         // All filters are on the stack at this point, and the filter source frame is bound:
         // therefore, even if the bounds to non intersect the filter frame, the filter
         // is still applied and any filter padding that is in the frame is rendered correctly.
+
+        let bounds;
+
+        // If cullArea is set, we use this rectangle instead of the bounds of the object. The cullArea
+        // rectangle must completely contain the container and its children including filter padding.
+        if (this.cullArea)
+        {
+            // Transform the cullArea rectangle to world space.
+            bounds = tempRect.copyFrom(this.cullArea);
+
+            const wt = this.worldTransform;
+            const lt = tempPoints[0];
+            const lb = tempPoints[1];
+            const rt = tempPoints[2];
+            const rb = tempPoints[3];
+
+            lt.set(bounds.left, bounds.top);
+            lb.set(bounds.left, bounds.bottom);
+            rt.set(bounds.right, bounds.top);
+            rb.set(bounds.right, bounds.bottom);
+
+            wt.apply(lt, lt);
+            wt.apply(lb, lb);
+            wt.apply(rt, rt);
+            wt.apply(rb, rb);
+
+            const x0 = Math.min(lt.x, lb.x, rt.x, rb.x);
+            const y0 = Math.min(lt.y, lb.y, rt.y, rb.y);
+            const x1 = Math.max(lt.x, lb.x, rt.x, rb.x);
+            const y1 = Math.max(lt.y, lb.y, rt.y, rb.y);
+
+            bounds.x = x0;
+            bounds.y = y0;
+            bounds.width = x1 - x0;
+            bounds.height = y1 - y0;
```

Comment:
Not really. I copied this code from `FilterSystem#transformAABB`. There's also `Bounds#addFrameMatrix`, but that's even longer.

Reply:
What I mean to say is that Container seems the the wrong place for this math. I see that it's very similar, if not identical to transformAABB, let's move that somewhere else (like to Rectangle or a static function in math package) so we aren't repeating.

Reply:
Sure. I was thinking instead of computing the bounds of the transformed cullArea and checking that for intersection we should do the exact intersection test of the transformed cullArea and the source frame to improve culling if rotation is present in the world transform. We could add a second parameter to `Rectangle#intersects`, the transform of the other rectangle, or add a new function `Rectangle#intersectsTransformed(other: Rectangle, matrix: Matrix)`.

Reply:
Sure, whatever keeps from overloading Container

Reply:
I added the transform parameter to `Rectangle#intersects`. If you prefer a separate function, let me know.